Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debian:10
+
+RUN true \
+    && apt-get update \
+    && apt-get install git curl openssh-client build-essential libssl-dev pkg-config -y \
+    && mkdir -p -m 0600 ~/.ssh \
+    && ssh-keyscan github.com >> ~/.ssh/known_hosts \
+    && curl https://sh.rustup.rs -o rustup \
+    && chmod +x rustup \
+    && ./rustup -y \
+    && git clone https://github.com/klausi/mastodon-twitter-sync.git \
```

Comment:
do we need to clone again? If people already clone this repository to have the Dockerfile locally - then they also have all the files already and we just need to mount them into the container?

Reply:
We can do that, it's just that they have to run the container from an empty directory afterwards because otherwise we get a problem with mounting the directory into the container to persist the data.

Reply:
Dockerfile and readme are updated and more streamlined now.